src/components/Settings/Settings.tsx

```diff
@@ -1,11 +1,22 @@
-import React from "react";
+import React, { useState } from "react";
 import { useNavigate } from "react-router-dom";
 import { XMarkIcon } from "@heroicons/react/24/solid";
 import { Input } from "@material-tailwind/react";
 
 function Settings() {
   const navigateTo = useNavigate();
 
+  const [deviceID, setDeviceID] = useState("Default Device ID");
+
+  const [deviceName, setDeviceName] = useState("Mr. Meshtastic");
+  const [deviceNickname, setDeviceNickname] = useState("MRM");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const sherryTest = { deviceID, deviceNickname, deviceName };
```

Comment:
Might want to switch the variable name `sherryTest` to `deviceInfoTest` if we're keeping the log?

Reply:
Meant to get rid of that, good catch!